scalameta/trees/shared/src/main/scala/scala/meta/internal/prettyprinters/TreeSyntax.scala

```diff
@@ -982,10 +1115,16 @@ object TreeSyntax {
       s("(", r(params, ", "), ")")
     }
     implicit def syntaxParamss: Syntax[List[List[Term.Param]]] = Syntax { paramss =>
+      def usingImplicit(params: List[Term.Param]): Show.Result = {
+        if (params.exists(_.mods.exists(_.is[Mod.Using])))
+          s("using ", r(params, ", "))
```

Comment:
Should it not be similar to implicit like:
`w("using ", r(params, ", "), params.exists(_.mods.exists(_.is[Mod.Using])))`
?

Reply:
it doesn't matter, we already did this check line above and we know using is there